certification/pyxis/builder.go

```diff
@@ -150,43 +134,39 @@ func (b *certificationInputBuilder) storeRPMManifest(filepath string) error {
 		)
 	}
 
-	var manifest RPMManifest
-	err = json.Unmarshal(bytes, &manifest)
+	err = json.Unmarshal(bytes, &submission)
 	if err != nil {
 		return fmt.Errorf(
 			"%w: data for the %s appears to be malformed: %s",
 			errors.ErrSubmittingToPyxis,
-			"rpm manifest",
+			"rpm manifest", // USE REFLECTION - Worth it?
```

Comment:
Nope! 😆

Reply:
Glad I put that in there! I'll take that out.